src/features/chat/Chat.tsx

```diff
@@ -0,0 +1,234 @@
+import React, { useEffect, useState, useRef, KeyboardEvent } from 'react'
+// import { useHistory } from 'react-router'
+import cn from 'classnames'
+import routes from '../../common/constants/routes.json'
+import { useHistory } from 'react-router-dom'
+import { ChatItem, TChatItemProps } from './ChatItem'
+import { ChatMessage } from './ChatMessage'
+import { UserAvatar } from './components/UserAvatar'
+import editIcon from '../../common/assets/icons/ico-edit.svg'
+import chatMenuIcon from '../../common/assets/icons/ico-chat-menu.svg'
+import msgEmojiIcon from '../../common/assets/icons/ico-chatmsg-emoji.svg'
+import msgAttachIcon from '../../common/assets/icons/ico-chatmsg-link.svg'
+import msgSendIcon from '../../common/assets/icons/ico-chatmsg-send.svg'
+import chatCloseIcon from '../../common/assets/icons/ico-close2.svg'
+import styles from './Chat.module.css'
+import { mockChats, curUser } from './mock-data'
+import { TChatUser } from './common'
+
+export default function Chat(): JSX.Element {
```

Comment:
Could you keep the same styling for this?
Sometimes you are using function and sometimes you are using const. I think you can keep the same styling using const(arrow function for creating component.)

Reply:
Updated